docs/user-guide/service-accounts.md

```diff
@@ -154,7 +154,7 @@ Automated version using json and the jq utility:
 kubectl get serviceaccounts default -o json |
      jq  'del(.metadata.resourceVersion)'|
      jq 'setpath(["imagePullSecrets"];[{"name":"myregistrykey"}])' |
-     kubectl replace serviceaccount default -f -
+     kubectl apply -f -
```

Comment:
it could actually be far simpler:
```
kubectl get serviceaccounts default -o json |
  jq '.imagePullSecrets += [{"name":"myregistrykey"}]' |
  kubectl replace -f -
```